scripts/do_benchmarks.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+import datetime
+import os
+import shlex
+import subprocess
+
+from fabric import Connection
+
+################################################################################
+# Constants
+################################################################################
+
+BENCH_HOST = "10.81.18.119"
```

Comment:
I am not sure checking in stuff that can only work on your machine is useful, or maybe I'm missing out on how this can be generalised?

Reply:
This is the internal IP for the Morello hybrid box we have at King's, so actually this only works on the King's internal network. I can replace the constant value with an environment variable, if you think it's more useful that way.

Reply:
My point is: is checking in this file is this repo the right thing to do? Is it an internal benchmarking thing that should be elsewhere or ... ?

Reply:
I would at it here for simplicity, and perhaps move `BENCH_{HOST, USER}` to read from environment variables. I would also not be against just leaving it un-pushed, but I think it's useful to have somewhere for ease and reproducibility, and I can't think of where elsewhere than this repo.

Reply:
OK, env vars work for me.

Reply:
Done in https://github.com/capablevms/CHERI-ELF-comp/pull/41/commits/cbbd6d1df0581b43aa4d28f6616bbbcef2092117.